herbert_core/admin/@opt_config_manager/opt_config_manager.m

```diff
@@ -58,17 +61,20 @@
         %
         this_pc_memory_ = 8*1024*1024*1024; %default PC memory is 8Gb
     end
+    properties(Constant,Access=private)
+        % different pc types, we have optimizezed Horace/Herbert for.         
+        pc_types_ = {'win_small','win_large','a_mac',...
+            'unix_small','unix_large',...
+            'idaaas_small','idaaas_large', 'jenkins_win', 'jenkins_unix'};
+        known_pc_types_ = containers.Map(opt_config_manager.pc_types_, ...
+            opt_config_manager.pc_types_) ;        
```

Comment:
this appears to be a map to itself

Reply:
yes, Better solution exist but not worth spending time on it.